src/material/schematics/ng-generate/mdc-migration/rules/template-migration.ts

```diff
@@ -0,0 +1,56 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {Migration, ResolvedResource} from '@angular/cdk/schematics';
+import {SchematicContext} from '@angular-devkit/schematics';
+import {StyleMigrator} from './style-migrator';
+import * as compiler from '@angular/compiler';
```

Comment:
It's super nice that this works. In framwork we needed a huge, painful workaround for this since schematics are CJS, but Angular packages are strict ESM. But here it works out nicely because we've set up bundling for the Sass AST, so it just works out nicely for the compiler as well! 😄